strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/artifact/coordinates/PypiArtifactCoordinates.java

```diff
@@ -115,6 +115,17 @@ public PypiArtifactCoordinates(String distribution,
         setPackaging(packaging);
     }
 
+    public PypiArtifactCoordinates()
+    {
+        this(DISTRIBUTION,
+             VERSION,
+             "",
```

Comment:
Also, I wanted to ask, why is the third parameter a blank string? Shouldn't it be like this:
```
    public PypiArtifactCoordinates()
    {
        this(DISTRIBUTION,
             VERSION,
             BUILD,
             LANGUAGE_IMPLEMENTATION_VERSION,
             ABI,
             PLATFORM,
             PACKAGING);
    }
```

And another cosmetic change, if you don't mind, could you please move the empty constructor above the other one? It's sort of a convention we have. Thanks! :)

Reply:
It should be empty or be a number but BUILD=="build" (not a number :))

Reply:
maybe the BUILD variable should be changed?

Reply:
These constructors look like crap. This is actually not how they should work. I believe the initial implementation, (now that I look at it more closely), is mistaken.

If you have a look at the `MavenArtifactCoordinates`, `NpmArtifactCoordinates`, or `NugetArtifactCoordinates`, you'll see what I mean. Each of the setter methods should actually be invoking `setCoordinate`.

Would you mind fixing this as well? I know it isn't part of your changes at all, but this seems to be quite mistaken and misleading.

Reply:
ok, no problem :)